backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/utils/VmCpuCountHelper.java

```diff
@@ -169,16 +169,20 @@ public static boolean validateCpuCounts(VM vm, Version compatibilityVersion, Arc
         return true;
     }
 
-    public static boolean isAutoPinning(VM vm) {
+    public static boolean isNumaAutoPinning(VM vm) {
```

Comment:
as it seems we'll want to have NUMA pinning in other cases and `isNumaAutoPinning` doesn't seems to reflect what we actually check here, how about `isResizeAndPinPolicy`?